instrumentation/kubernetes/src/main/java/io/opentelemetry/auto/instrumentation/kubernetes/KubernetesClientInstrumentation.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.auto.instrumentation.kubernetes;
+
+import static java.util.Collections.singletonMap;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+import com.google.auto.service.AutoService;
+import io.kubernetes.client.openapi.ApiClient;
+import io.opentelemetry.auto.tooling.Instrumenter;
+import java.util.List;
+import java.util.Map;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+import net.bytebuddy.matcher.ElementMatchers;
+import okhttp3.Interceptor;
+
+/**
+ * @author zuoxiu.jm
+ * @version : KubernetesClientInstrumentation.java, v 0.1 2020年07月08日 4:35 PM zuoxiu.jm Exp $
+ */
+@AutoService(Instrumenter.class)
+public class KubernetesClientInstrumentation extends Instrumenter.Default {
+
+  public KubernetesClientInstrumentation() {
+    super("kubernetes");
+  }
+
+  @Override
+  public ElementMatcher<TypeDescription> typeMatcher() {
```

Comment:
Please add class loader matcher as well. There are plenty of examples in other instrumenters.

Reply:
i wrapped the name matcher w/ `extendsClass(..)`, is that what you're expecting?

Reply:
No, I mean
```
@Override
  public ElementMatcher<ClassLoader> classLoaderMatcher() {
    return hasClassesNamed("io.kubernetes.client.openapi.ApiClient");
  }
```